code/modules/clothing/masks/breath.dm

```diff
@@ -62,7 +62,68 @@
 	desc = "A close-fitting cap that covers the top, back, and sides of the head. Can also be adjusted to cover the lower part of the face so it keeps the user warm in harsh conditions."
 	icon_state = "coif"
 	item_state = "coif"
-	flags_inventory = COVERMOUTH|ALLOWREBREATH
 	flags_inv_hide = HIDEALLHAIR|HIDEEARS
 	flags_cold_protection = HEAD
 	min_cold_protection_temperature = ICE_PLANET_MIN_COLD_PROTECTION_TEMPERATURE
+
+/obj/item/clothing/mask/rebreather/bandanna
```

Comment:
Why is this a child of the `rebreather`?

Reply:
I mean, should it NOT be? I did it because the Rebreather has all the same flags I want

Reply:
Yeah that's not how that works

Reply:
Well, the question is on if it works like a rebreather. I was assuming it didn't.